src/app/store.ts

```diff
@@ -1,6 +1,7 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { currentTodoSlice, filterSlice, todosSlice } from '../features';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(currentTodoSlice, filterSlice, todosSlice);
```

Comment:
Change `combineSlices` to `combineReducers` to correctly combine the slices into a root reducer.